Darwin/StitchCore/StitchCoreTests/Core/Auth/StitchAuthDelegateIntegrationTests.swift

```diff
@@ -33,7 +33,7 @@ class StitchAuthDelegateIntTests: BaseStitchIntTestCocoaTouch {
     }
 
     func waitForDelegate(work: DispatchGroup) {
-        guard work.wait(timeout: .now() + 10) == .success else {
+        guard work.wait(timeout: .now() + 20) == .success else {
```

Comment:
timeout is an absolute time?

Reply:
[q] Am I reading the docs right in that this is nanosecond precision? if it is, we should probably wait at least a second.

Reply:
updated